<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureEditorDockWidget</class>
 <widget class="QDockWidget" name="TextureEditorDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextureEditorDockWidget</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="mainHorizontalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <widget class="QMdiArea" name="mdiArea">
        <property name="viewMode">
         <enum>QMdiArea::TabbedView</enum>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>true</bool>
        </property>
        <property name="tabsMovable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="propertiesVerticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="texturePreviewFrame">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>256</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="propertiesFrame">
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QLabel" name="formatValueLabel">
             <property name="text">
              <string>RGBA 8</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="formatLabel">
             <property name="text">
              <string>Format:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="resolutionLabel">
             <property name="text">
              <string>Resolution:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="resolutionValueLabel">
             <property name="text">
              <string>256 x 256</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="propertiesVerticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionTest1">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Toolbar/File/OpenApp24.png</normaloff>:/Toolbar/File/OpenApp24.png</iconset>
   </property>
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
  <action name="actionTest1_2">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Toolbar/File/SaveScene24.png</normaloff>:/Toolbar/File/SaveScene24.png</iconset>
   </property>
   <property name="text">
    <string>Test1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
